web/filament-js/jsbindings.cpp

```diff
@@ -1411,7 +1411,10 @@ class_<MaterialInstance>("MaterialInstance")
     .function("setStencilWriteMask", EMBIND_LAMBDA(void,
             (MaterialInstance* self, uint8_t writeMask), {
                 self->setStencilWriteMask(writeMask, backend::StencilFace::FRONT_AND_BACK);
-            }), allow_raw_pointers());
+            }), allow_raw_pointers())
+    .function("setFrontFaceWindingInverted", &MaterialInstance::setFrontFaceWindingInverted)
+    .function("isFrontFaceWindingInverted", &MaterialInstance::isFrontFaceWindingInverted);
```

Comment:
we don't need to remove the View's version? were they missing?

Reply:
yes, the View methods were missing.